### PR TITLE
[OSDOCS-6756]:Persistent Storage using Amazon EBS Prebuilt Storage Classes

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-6756
Link to docs preview:

ROSA
https://62104--docspreview.netlify.app/openshift-rosa/latest/storage/persistent_storage/persistent-storage-aws.html
OSD
https://62104--docspreview.netlify.app/openshift-dedicated/latest/storage/persistent_storage/persistent-storage-aws.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by 0kashi** [comment]
lgtm.

Just one comment.  In that paragraph one time we are referring to EBS as "AWS EBS" and then later to "Amazon EBS".  I think for consistency we should be using "Amazon EBS".

**Comment by mletalie** [comment]
Good catch, @0kashi , will update based on your suggestion. I appreciate it.

**Comment by mletalie** [comment]
image

File: storage/persistent_storage/persistent-storage-aws.adoc [hunk 1]
```diff
@@ -34,22 +34,11 @@ You can provision your {product-title} cluster with persistent storage by using
...
-{product-title} defaults to using an in-tree, or non-Container Storage Interface (CSI) plug-in to provision AWS EBS storage. In future {product-title} versions, volumes provisioned using existing in-tree plug-ins are planned for migration to their equivalent CSI driver. 
-
-CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes.
-ifndef::openshift-rosa,openshift-dedicated[]
-For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
-endif::openshift-rosa,openshift-dedicated[]
-
-After full migration, in-tree plugins will eventually be removed in future versions of {product-title}.
-====
+By default, newly created clusters using ROSA version 4.10 and later use gp3 storage and the Amazon Elastic Block Store (Amazon EBS) Container Storage Interface (CSI) driver.
```
Comment (GroceryBoyJr) [inline storage/persistent_storage/persistent-storage-aws.adoc:41]:
```suggestion
By default, newly created clusters using {product-title} version 4.10 and later use gp3 storage and the Amazon Elastic Block Store (Amazon EBS) Container Storage Interface (CSI) driver.
```

Comment (GroceryBoyJr) [reply storage/persistent_storage/persistent-storage-aws.adoc:41]: @mletalie I think? Ponder on it.

**Review by GroceryBoyJr — COMMENTED** [review]
Only one small nit please sir.

File: storage/persistent_storage/persistent-storage-aws.adoc (hunk 1 above)
Comment (mletalie) [reply storage/persistent_storage/persistent-storage-aws.adoc:41]: Hello @GroceryBoyJr, eh.... good point. Let me look into it. Appreciate it!

**Comment by mletalie** [comment]
Hello @GroceryBoyJr,
    Thanks again for catching that product call out  issue. If you could take another look  to make sure everything is okay I would appreciate it. Thanks.

**Comment by xueli181114** [comment]
LGTM

**Comment by mburke5678** [comment]
@mletalie I just want to make sure this change applies to OCP also. The bug -- and your previews -- call out ROSA and OSD specifically. 
If so, can we drop the _version 4.10 and later_ because we don't update docs prior to 4.10 (so it applies to all supported versions) and ROSA and OSD don't use that versioning system (so it is irrelevant).

**Pull request merged**